Permanent/Views/Cells/FileDetailsCollectionViewCells/FileDetailsMapViewCellCollectionViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  FileDetailsMapViewCellCollectionViewCell.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.03.2021.
+//
+
+import UIKit
+import MapKit
+
+class FileDetailsMapViewCellCollectionViewCell: UICollectionViewCell {
+
+    static let identifier = "FileDetailsMapViewCellCollectionViewCell"
+    let regionInMeters: Double = 5000
+    
+    @IBOutlet weak var titleLabelField: UILabel!
+    @IBOutlet weak var detailsTextField: UITextField!
```

Comment:
This TextField can be a Label, since it's not editable

Reply:
I will change detailsTextField to detailsLabelField.